### PR TITLE
Add missing import

### PR DESCRIPTION
**Pull request by llazzaro**
EAPOL was missing, a possible cause was import *.

to reproduce try to execute:

`from scapy.contrib.wpa_eapol import WPA_key
`

### CODE REVIEW

File: scapy/contrib/wpa_eapol.py [hunk 1]
```diff
@@ -27,7 +29,7 @@ class WPA_key(Packet):
                     StrFixedLenField("replay_counter", "", 8),
                     StrFixedLenField("nonce", "", 32),
                     StrFixedLenField("key_iv", "", 16),
-                    StrFixedLenField("wpa_key_rsc", "", 8), 
+                    StrFixedLenField("wpa_key_rsc", "", 8),
```
Comment (p-l-) [inline scapy/contrib/wpa_eapol.py]: Please remove this useless change.

File: scapy/contrib/wpa_eapol.py [hunk 2]
```diff
@@ -41,6 +43,6 @@ def answers(self, other):
         if isinstance(other,WPA_key):
                return 1
         return 0
-             
+
```
Comment (p-l-) [inline scapy/contrib/wpa_eapol.py]: Please remove this useless change.

Comment (llazzaro) [reply scapy/contrib/wpa_eapol.py]: ok! I understand the policy of keeping the history clean.
But usually white spaces should be removed.

**Comment by llazzaro** [comment]
I will open another pull request.

**Comment by p-l-** [comment]
@llazzaro you can just `git commit --amend --no-edit` your changes, then `git push -f` them.

**Comment by llazzaro** [comment]
@p-l- oh! I didn't know that I was able to do push -f! I created another pr. hope it's ok. thanks!

**Pull request closed without merging**